tests/hypothesis_strategies/targets.py

```diff
@@ -0,0 +1,34 @@
+"""Hypothesis strategies for targets."""
+
+import hypothesis.strategies as st
+from attrs import NOTHING
+
+from baybe.targets.enum import TargetMode
+from baybe.targets.numerical import _VALID_TRANSFORM_MODES, NumericalTarget
+
+from .utils import interval
+
+target_name = st.text(min_size=1)
+"""A strategy that generates target names."""
+
+
+@st.composite
+def numerical_target(draw: st.DrawFn):
+    """Generate class:`baybe.targets.numerical.NumericalTarget`."""
```

Comment:
Please verify that this link is created properly as I think that there is at least one : missing

Reply:
done